src/net.cpp

```diff
@@ -2909,12 +2915,13 @@ void CConnman::OpenNetworkConnection(const CAddress& addrConnect, bool fCountFai
         return;
 
     CNode* pnode = ConnectNode(addrConnect, pszDest, fCountFailure, conn_type, use_v2transport);
-
     if (!pnode)
         return;
+
     pnode->grantOutbound = std::move(grant_outbound);
 
-    m_msgproc->InitializeNode(*pnode, nLocalServices);
+    ServiceFlags nodeServices = nLocalServices;
+    m_msgproc->InitializeNode(*pnode, nodeServices);
```

Comment:
I don't think so because it's expected to not modify `nLocalServices`. Doesn't it?

```cpp
    /**
     * Services this node offers.
     *
     * This data is replicated in each Peer instance we create.
     *
     * This data is not marked const, but after being set it should not
     * change.
     *
     * \sa Peer::our_services
     */
    ServiceFlags nLocalServices;
```

Reply:
What does `\sa` mean?

Reply:
a doxygen command :)
https://www.doxygen.nl/manual/commands.html#cmdsa